### DISCUSSION TITLE
Filter Object "Builder"

### DISCUSSION THREAD

**Discussion by Michael-Nussbaumer** [directus/directus#12975] [Feature Requests]
When I currently need some complexer filters, I like to use the Layout view and click together the filter I want to have, then I use the network tab of the dev console to get the URL and extract the Filter Object from the URL there. 

The Idea would be that there is maybe a module or something where u can also click together some filters like we use it in many places already, which gives you the Filter Object for copy&paste. Especially for more complex combinations, this would be a breeze.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
